org.pyzo.pyzo.json

```diff
@@ -0,0 +1,74 @@
+{
+  "app-id" : "org.pyzo.pyzo",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.10",
+  "sdk" : "org.kde.Sdk",
+  "command" : "pyzo",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=host"
+  ],
+  "modules" : [
+    {
+      "name" : "pyqt5",
+      "buildsystem" : "simple",
+      "build-commands" : [
+          "pip3 install --no-index --find-links \"file://${PWD}\" --prefix=/app PyQt5==5.9.2"
+      ],
+      "sources" : [
+          {
+              "type" : "file",
+              "url" : "https://files.pythonhosted.org/packages/3a/c6/26270f5550f00920045c2f0b222a7d03d7a64382825c68bf0bb1a51d854c/PyQt5-5.9.2-5.9.3-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl",
```

Comment:
Avoid binaries, build from source. (Some other packages on flathub do this if you search for `pyqt`).